backend/src/LibRealExecution/RealExecution.fs

```diff
@@ -84,7 +84,7 @@ let createState
 type ExecutionReason =
   /// The first time a trace is executed. This means more data should be stored and
   /// more users notified.
-  | InitialExecution of HandlerDesc * RT.Dval
+  | InitialExecution of HandlerDesc * string * RT.Dval
```

Comment:
```suggestion
  | InitialExecution of HandlerDesc * varname: string * inputVar: RT.Dval
```
I'm not sure what the string is for here - worth labeling the tuple parts

Reply:
Done.